package.json

```diff
@@ -117,34 +118,57 @@
     "eslint-plugin-import": "2.26.0",
     "eslint-plugin-sonarjs": "0.16.0",
     "eslint-plugin-unicorn": "44.0.2",
-    "esm": "^3.2.25",
-    "dotenv": "16.0.3",
-    "husky": "^8.0.0",
-    "isomorphic-fetch": "^3.0.0",
-    "jest": "^29.1.1",
-    "jest-date-mock": "^1.0.8",
-    "jest-environment-jsdom": "^29.1.1",
+    "esm": "3.2.25",
+    "husky": "8.0.2",
+    "isomorphic-fetch": "3.0.0",
+    "jest": "29.3.0",
+    "jest-date-mock": "1.0.8",
+    "jest-environment-jsdom": "29.3.0",
     "jest-sonar": "0.2.12",
     "jscpd": "3.5.1",
-    "lint-staged": "^13.0.1",
+    "lint-staged": "13.0.3",
     "madge": "5.0.1",
-    "msw": "^0.47.3",
-    "prettier": "^2.6.2",
-    "rollup": "^2.75.3",
-    "rollup-plugin-brotli": "^3.1.0",
-    "rollup-plugin-copy": "^3.4.0",
-    "rollup-plugin-filesize": "^9.1.2",
-    "rollup-plugin-gzip": "^3.0.1",
-    "rollup-plugin-node-builtins": "^2.1.2",
-    "rollup-plugin-node-globals": "^1.4.0",
-    "rollup-plugin-terser": "^7.0.2",
-    "rollup-plugin-visualizer": "^5.6.0",
+    "msw": "0.47.4",
+    "prettier": "2.7.1",
+    "rollup": "2.79.1",
+    "rollup-plugin-copy": "3.4.0",
+    "rollup-plugin-filesize": "9.1.2",
     "rollup-plugin-generate-html-template": "1.7.0",
     "rollup-plugin-livereload": "2.0.5",
+    "rollup-plugin-node-builtins": "2.1.2",
+    "rollup-plugin-node-globals": "1.4.0",
     "rollup-plugin-serve": "2.0.1",
+    "rollup-plugin-terser": "7.0.2",
+    "rollup-plugin-visualizer": "5.8.3",
     "size-limit": "^7.0.8",
     "standard-version": "9.5.0"
   },
+  "overrides": {
+    "@xmldom/xmldom": "0.8.5",
+    "dot-prop": "5.3.0",
+    "semver": "7.3.8",
+    "semver-regex": "3.1.4",
+    "trim-newlines": "3.0.1"
+  },
+  "browserslist": {
```

Comment:
How is this info consumed into rollup, and the modern SDK is of a smaller size?

Reply:
Rollup uses babel to transpile and build the code output. Babel uses presets to define what it will transpile to. The preset env respects the browserlist consiguration based on selected browserlist env.

More tools respect browserlist in the same way see https://github.com/browserslist/browserslist top of the file

You can see what is the full list of browsers that will be supported be npm run check:support and test configurations at https://browsersl.ist/ right side of the page